src/NuGet.Clients/NuGet.PackageManagement.PowerShellCmdlets/Cmdlets/GetProjectCommand.cs

```diff
@@ -50,24 +52,26 @@ protected override void ProcessRecordCore()
 
             if (All.IsPresent)
             {
-                var projects = EnvDTESolutionUtility.GetAllEnvDTEProjects(DTE);
+                VsSolutionManager.EnsureSolutionIsLoaded();
+                var projects = VsSolutionManager.GetAllVsProjectAdapters().Select(p => p.Project);
```

Comment:
Current contract is to return entire DTE project:
```cs
[OutputType(typeof(EnvDTE.Project))]
```
I don't think we should break it.